### ISSUE TITLE
Inference of phenotype-relevant transcriptional regulatory networks elucidates cancer type-specific regulatory mechanisms in a pan-cancer study

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#10]
## TL;DR

 This paper develops a method called InPheRNo to identify “phenotype-relevant” transcriptional regulatory networks (TRNs).

### Paper Link

https://www.nature.com/articles/s41540-021-00169-7

### Author/Institution

Amin Emad (McGill University), Saurabh Sinha (University of Illinois, )

## Overview

Reconstruction of transcriptional regulatory networks (TRNs) is a powerful approach to unravel the gene expression programs involved in healthy and disease states of a cell. However, these networks are usually reconstructed independent of the phenotypic (or clinical) properties of the samples. They may therefore confound regulatory mechanisms that are specifically related to a phenotypic property. 

The InPheRNo method  is used to identify “phenotype-relevant” TRNs. The method is based on a probabilistic graphical model that models the simultaneous effects of multiple transcription factors (TFs) on their target genes and the statistical relationship between the target genes’ expression and the phenotype.

## Cite

Emad, A., Sinha, S. Inference of phenotype-relevant transcriptional regulatory networks elucidates cancer type-specific regulatory mechanisms in a pan-cancer study. npj Syst Biol Appl 7, 9 (2021). https://doi.org/10.1038/s41540-021-00169-7

## Comments

Implementations of InPheRNo (python): https://github.com/KnowEnG/InPheRNo 
Implementations of simplified-InPheRNo (python): https://github.com/KnowEnG/Simplified-InPheRNo
